src/main/java/com/synopsys/integration/detect/lifecycle/run/DetectRun.java

```diff
@@ -127,6 +127,9 @@ public void run(BootSingletons bootSingletons) {
                     intelligentModeSteps.runOffline(nameVersion, universalToolsResult.getDockerTargetData(), bdio);
                 }
             }
+            if(autonomousManager.getAutonomousScanEnabled()) {
```

Comment:
Moved the order for saving autonomous file to get the binary scan targets passed by user.